doc/source/getting-started/index.rst

```diff
@@ -0,0 +1,35 @@
+.. title:: Getting Started
+
+Getting started
+===============
+
+Click the following cards to learn how to set up the available pre-commit hooks:
```

Comment:
```suggestion
Learn how to set up the available pre-commit hooks:
```
We don't usually tell them to click the following cards before using a grid. We assume that our users understand this.